src/js/ttsHookup.js

```diff
@@ -74,4 +91,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         speakFn(msg);
     };
 
+    gpii.firstDiscovery.tts.fdHookup.speakPanelInstructions = function (that, speakFn) {
+        var msg = that.panels.eq(that.model.currentPanelNum - 1).find(that.options.panelInstructionsSelector).text();
```

Comment:
Duplicated code as line 89 to retrieve the instruction. You may want to create a separate function for it.